integration/cluster.go

```diff
@@ -299,6 +321,11 @@ func (c *cluster) waitLeader(t *testing.T, membs []*member) int {
 	for lead == 0 || !possibleLead[lead] {
 		lead = 0
 		for _, m := range membs {
+			select {
+			case <-m.s.StopNotify():
```

Comment:
I think adding a bool would be more complicated-- member.Stop() would need to launch a goroutine to update a bool which has the same meaning as StopNotify() closed.